web/src/settings_bots.ts

```diff
@@ -207,6 +207,7 @@ export function add_a_new_bot(): void {
             ).val()!;
             $(`[name*='${CSS.escape(selected_bot)}']`).show();
         });
+        window.location.href = "/#organization/bots/your-bots";
```

Comment:
Can we use `toggler.goto` here?

Reply:
And I don't think this is being called after the bot is created, but the tab is changed after the modal to add a bot renders.

Reply:
I am not able to import `settings_panel_menu.js` in `web/src/settings_bots.ts` .
due to this error
```console
Could not find a declaration file for module './settings_panel_menu'. '/home/sujal/zulip/web/src/settings_panel_menu.js' implicitly has an 'any' type.ts(7016)
```

However, I have changed the location of `window.location.href = "/#organization/bots/your-bots";`  inside add bot function.